ontology/yaml/resources/units/units.yaml

```diff
@@ -514,4 +514,14 @@ volume:
     multiplier: 0.028316846592
     offset: 0
 volumetricconcentration:
-  parts_per_cubic_centimeter: STANDARD # can be approximated using Avogadro
\ No newline at end of file
+  parts_per_cubic_centimeter: STANDARD # can be approximated using Avogadro
+xvalue:
+  no_units: STANDARD
+yvalue:
+  no_units: STANDARD
+group: 
```

Comment:
group should not need units, since it is not a measurement.

Reply:
I don't understand this remark. group is a descriptor. Why should a descriptor have a unit?

Reply:
Damn right... Now, even I see it